test/verify/check-kdump

```diff
@@ -45,23 +65,26 @@ class KdumpHelpers(testlib.MachineCase):
 
             # wait for disconnect and then try connecting again
             b.switch_to_top()
-            with b.wait_timeout(60):
+            with b.wait_timeout(120):
```

Comment:
I saw this in a log, where it reaches 70% of the timeout. Of course I couldn't find the log itself.